v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb

```diff
@@ -439,8 +451,82 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pd.DataFrame(\n",
-    "    {\"metric name\": list(test_metrics.keys()), \"value\": list(test_metrics.values())}\n",
+    "def named_entity_recognition_report(predictions_file, test_data_file, use_str = True):\n",
```

Comment:
Possible to move this to a utils.py kind of file in this folder? @msp8955 wdyt?

Reply:
yeah similar to forecasting example having metrics_helper.py
https://github.com/Azure/azureml-examples/tree/main/v1/python-sdk/tutorials/automl-with-azureml/forecasting-bike-share

Reply:
Done